e2e/etcd_test.go

```diff
@@ -433,9 +433,9 @@ func (ep *etcdProcess) Stop() error {
 }
 
 func (ep *etcdProcess) waitReady() error {
-	readyStrs := []string{"enabled capabilities for version", "published"}
+	readyStrs := []string{"starting server...", "published"}
```

Comment:
No, that's not going to happen. The tests are meant for the current version of etcd; tests are only backported when it's convenient. It's too much trouble to worry about whether changes to the current tests break on older versions of etcd and there's not enough CI resources to support it. Are there any go projects with test suites which are backwards compatible like that?